frontends/precinct-scanner/src/components/set_mark_thresholds_modal.tsx

```diff
@@ -0,0 +1,237 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import { MarkThresholds } from '@votingworks/types';
+import {
+  Button,
+  LinkButton,
+  Loading,
+  Modal,
+  Prose,
+  Text,
+} from '@votingworks/ui';
+import { assert, throwIllegalValue } from '@votingworks/utils';
+
+export interface Props {
+  onClose: () => void;
+  markThresholds?: MarkThresholds;
+  markThresholdOverrides?: MarkThresholds;
+  setMarkThresholdOverrides: (markThresholds?: MarkThresholds) => Promise<void>;
+}
+
+const ThresholdColumns = styled.div`
+  display: flex;
+  flex-direction: row;
+  > div {
+    flex: 1;
+  }
+`;
+
+enum ModalState {
+  SAVING = 'saving',
+  RESET_THRESHOLDS = 'reset_thresholds',
+  SET_THRESHOLDS = 'set_thresholds',
+  CONFIRM_INTENT = 'confirm_intent',
+  ERROR = 'error',
+}
+
+export const DefaultMarkThresholds: Readonly<MarkThresholds> = {
+  marginal: 0.17,
+  definite: 0.25,
+};
+
+export function SetMarkThresholdsModal({
```

Comment:
Definitely a post-m14 thought that doesn't need to be addressed now! Just noting that we could move this component to a shared location since it's identical across VxScan and VxCentralScan (I know that's going to change a bit once we account for the lack of a keyboard on VxScan)